libvast/include/vast/system/spawn_arguments.hpp

```diff
@@ -67,7 +67,7 @@ caf::expected<expression>
 normalized_and_validated(std::vector<std::string>::const_iterator begin,
                          std::vector<std::string>::const_iterator end);
 
-caf::expected<expression> get_expression(const spawn_arguments& args);
+caf::expected<expression> get_expression(const std::vector<std::string>& args);
```

Comment:
The spawn arguments still include an optional expression. That shouldn't exist anymore after this change.